nomad-gcp/modules/nomad-server-gcp/server.tf

```diff
@@ -4,7 +4,7 @@ data "google_compute_image" "machine_image" {
 }
 
 data "google_compute_subnetwork" "nomad" {
-  name   = var.subnetwork ? var.subnetwork : var.network
+  name   = var.subnetwork != "" ? var.subnetwork : var.network
```

Comment:
Syntax error that was only picked up by `terraform test`, but not `terraform validate`...